core/meterer/meterer.go

```diff
@@ -12,14 +12,6 @@ import (
 
 // Config contains network parameters that should be published on-chain. We currently configure these params through disperser env vars.
 type Config struct {
```

Comment:
It looks like it needs to wait for 1 hour to load the onchain payment state (in `Start` L59). 
Should it refresh it immediately on start?

Reply:
sounds good, I added a separate refresh before the ticker loop